### ISSUE TITLE
Should we sideload the full size image?

### ISSUE THREAD

**Issue by MichaelArestad** [MichaelArestad/Press-This#82]
> What's not as cool is teh full size sideloaded image. I'd change that every post if I was using this, to at least use "large"

via @krogsgard

**Comment by stephdau** [comment]
Is that suggestion to just apply a standard wp class name to the img tag we insert in the editor?

**Comment by krogsgard** [comment]
No, I mean to not insert the full size image. If it's coming from another
website it could be quite large. Like 2000px or something. Once it
sideloads and inserts, I'd insert it at "large" bc that should reasonably
be the largest inserted image for any WordPress site, per their media
settings.

Brian Krogsgard
Krogsgard@gmail.com
Krogsgard.com
251.709.2224

**Comment by DrewAPicture** [comment]
I agree with @krogsgard. Sideload the full-size image, sure, but then only insert the 'large' image size. We'd probably have to make that image size filterable though.
